Volontulo/app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,15 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
+    <!-- Debug Drawer permissions -->
```

Comment:
If these permissions are only for debug version, maybe better place is for that `debug` flavor.